brand/src/bin/brand.rs

```diff
@@ -282,6 +282,16 @@ fn cmd_install(
         extra.unpack(&root)?;
     }
 
+    /*
+     * Copy in configuration files from the global zone.
+     */
+    for cf in ["default/init"] {
+        let src = format!("/etc/{cf}");
+        println!("INFO: omicron: copying {src}...");
+        let dst = s.zonerootpath(&["etc", cf]);
+        std::fs::copy(src, dst)?;
+    }
```

Comment:
I would defensively unlink the target before trying to make the copy, even though we do not currently expect that this will be a symlink or whatever else:

```suggestion
        std::fs::remove_file(&dst).ok();
        std::fs::copy(src, dst)?;
    }
```